elkscmd/basic/host-ibmpc.c

```diff
@@ -0,0 +1,359 @@
+/*
+ * Architecture Specific routines for CGA
+ * Sep 2024 Takahiro Yamada
+ */
+#include <stdio.h>
+#include <stdlib.h>
+#if __ia16__
+#include <arch/io.h>
+#else
+#define inb(port)           0
+#define inw(port)           0
+#define outb(value,port)
+#define outw(value,port)
+#endif
+#include "host.h"
+#include "basic.h"
+
+#define VIDEO_05_G320x200   0x0005
+#define VIDEO_03_T80x25     0x0003
+
+extern FILE *outfile;
```

Comment:
Lets move this (and the same in host-stubs.c) into host.h, since now there are two host-xxx.c files using the extern.

Reply:
OK.
host-pc98.c also uses host.h but may be that is not big problem.

Reply:
Oh, is this mean only the extern FILE?
or is it also for the #define VIDEO?
(I thought it is also for the define.)

Reply:
> Oh, is this mean only the extern FILE?

I was only referring to the extern FILE *.